portfolio/src/main/webapp/index.html

```diff
@@ -8,20 +8,24 @@
   </head>
   <body>
     <div id="content">
-        <h1>Obafemi Beloved Elegbede's Portfolio</h1>
-        <div>    
-            <p>LinkedIn: <a href="https://linkedin.com/in/obafemie/">Obafemie</a></p>
-            <p>Github: <a href="https://github.com/Beloved1">Beloved1</a></p>
+        <div class="header">
+            <div class="social">
+                <a href="https://linkedin.com/in/obafemie/"><img src="images/linkedin (1).png" alt="" height="32px" width="32px"></a>
```

Comment:
This is a minor thing, but this file name with the (1) in it feels inconsistent with the other names. Thinking to a larger project, others contributing might wonder why it's there or think that there are multiple linkedin images.

Reply:
Thank you. I will change it now. It was there because I already had an image with that name